package.json

```diff
@@ -41,20 +56,34 @@
     "@rollup/plugin-json": "^4.1.0",
     "@rollup/plugin-node-resolve": "^13.3.0",
     "@rollup/plugin-replace": "^4.0.0",
-    "@types/bn.js": "^5.1.0",
+    "@rollup/plugin-typescript": "^8.5.0",
+    "@types/bn.js": "^5.1.1",
     "@types/debug": "^4.1.7",
-    "@types/lodash.clonedeep": "^4.5.6",
+    "@types/lodash.clonedeep": "^4.5.7",
     "@types/lodash.isequal": "^4.5.6",
     "@types/mime": "^2.0.3",
-    "@types/sinon": "^10.0.11",
+    "@types/sinon": "^10.0.13",
     "@types/tape": "^4.13.2",
+    "@typescript-eslint/eslint-plugin": "^5.37.0",
+    "@typescript-eslint/parser": "^5.37.0",
     "babel-plugin-module-resolver": "^4.1.0",
+    "depcheck": "^1.4.3",
+    "eslint": "^8.23.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-cypress": "^2.12.1",
+    "eslint-plugin-eslint-plugin": "^5.0.6",
+    "eslint-plugin-flowtype": "^8.0.3",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-json": "^3.1.0",
+    "eslint-plugin-markdown": "^3.0.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-sort-class-members": "^1.15.2",
     "gh-pages": "^4.0.0",
-    "lerna": "^5.1.8",
-    "nx": "^14.4.3",
-    "prettier": "^2.6.0",
+    "nx": "^14.7.5",
+    "prettier": "^2.7.1",
     "rimraf": "^3.0.2",
     "rollup": "^2.72.0",
+    "rollup-plugin-generate-package-json": "^3.2.0",
```

Comment:
We don't need this plugin since we export using the right extension with rollup (.cjs and .mjs)

Reply:
removed